books/free-programming-books-ar.md

```diff
@@ -110,7 +117,7 @@
 
 ### Security
 
-* [تأمين الشبكات اللاسلكية للمستخدم المنزلي](http://librebooks.org/secure-wireless-networks-for-home-users/)
+* [تأمين الشبكات اللاسلكية للمستخدم المنزلي](http://librebooks.org/secure-wireless-networks-for-home-users/) - Mohamed Adel (PDF)
```

Comment:
:information:source: librebook download target points to Dropbox.
:question: Could your guide act as front matter? https://mohamedation.github.io/securing-wifi/

Reply:
i could change the link to a github one (https://github.com/mohamedation/securing-wifi/raw/master/securing-wifi-Arabic-old.pdf) for the pdf
the online guide is just an updated version of the book/guide..i am not comfortable calling it a book and adding it to books section.. what do you think is the best way?

Reply:
Taking a look at other of our `books-*.md` files, you realize that guides / tutorials abording a homogeneous topic... (see CONTRIBUTING guidelines for more) fits well in our `books` genre :wink:

The little problem with storage sites backed by a frontend like librebooks (it collects links as us) is that can't be tracked. This is one of the reasons why we prefer the sites owned or under author supervision

As creator and following this guidelines, feel free to choose the source you mind is appropriate

Reply:
thank you very much for guiding me. i have replaced it with  https://mohamedation.github.io/securing-wifi/ and changed the format.